src/derive/curve.rs

```diff
@@ -411,6 +411,11 @@ macro_rules! new_curve_impl {
             const fn curve_constant_b() -> $base {
                 $name_affine::curve_constant_b()
             }
+
+            #[inline]
+            fn curve_constant_3b() -> $base {
+                $constant_b + $constant_b + $constant_b
+            }
```

Comment:
Why not having this as a `const`?? It's much better than compute 3 times in 3 places this value.

Reply:
ping @davidnevadoc

Reply:
Ohh yeye, there's no way. Unless you introduce a `lazy_static` call inside of the macro which creates the constant. This could be one approach. 
You can also leave an issue filled for this and we can explore it later!